xonsh/history.py

```diff
@@ -673,6 +653,9 @@ def _hist_parse_args(args):
     parser = _hist_create_parser()
     if not args:
         args = ['show']
+    elif args[0] in _HIST_SESSIONS:
+        print('Use the `show` subcommand to get history sessions.', file=sys.stderr)
+        raise SystemExit
```

Comment:
you're right it's a temporary hack mostly, i may change arg parsing a little.
(btw i think we should foolproof against SystemExit, many parsers raise this on bad arguments or when printing help msgs.)